src/main/java/core/basesyntax/service/operations/FruitOperation.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+
+public interface FruitOperation {
+    Integer NAME_INDEX = 1;
+    Integer QUANTITY_INDEX = 2;
+
+    void operation(String[] data);
+
+    default void increaseQuantity(String[] data) {
+        Fruit temporaryFruit = new Fruit(data[NAME_INDEX]);
+        Storage.FRUIT_STORAGE.put(temporaryFruit,
+                Storage.FRUIT_STORAGE.get(temporaryFruit) + Integer.parseInt(data[QUANTITY_INDEX]));
+    }
+
+    default void reduceQuantity(String[] data) {
+        Fruit temporaryFruit = new Fruit(data[NAME_INDEX]);
+        Storage.FRUIT_STORAGE.put(temporaryFruit,
+                Storage.FRUIT_STORAGE.get(temporaryFruit) - Integer.parseInt(data[QUANTITY_INDEX]));
+    }
+
+    default void createKeyValue(String[] data) {
+        Fruit temporaryFruit = new Fruit(data[NAME_INDEX]);
+        Storage.FRUIT_STORAGE.put(temporaryFruit, Integer.parseInt(data[QUANTITY_INDEX]));
+    }

```

Comment:
Interface is a feature for abstraction. But here, you have already implemented methods, so it's not about abstraction anymore. Also, if logic of this methods will be changed, you will refactor interface, not its implementation.

Reply:
Firstly, logic was, as you say (methods into classes), so I will backup it=)